spring-cloud-task-batch/src/main/java/org/springframework/cloud/task/batch/configuration/TaskBatchAutoConfiguration.java

```diff
@@ -32,11 +32,14 @@
 /**
  * Provides auto configuration for the {@link TaskBatchExecutionListener}.
  *
+ * The spring.cloud.task.batch.listener.enable is being deprecated,
```

Comment:
`is deprecated`...not `is being deprecated` (makes it sound like we're going to get to it later).